default.zshrc

```diff
@@ -13,4 +13,5 @@ alias la='ls -A'
 alias l='ls -CF'
 
 # Common environment variables
+AWS_SDK_LOAD_CONFIG=1
```

Comment:
You'll want to add this to your `~/.zshrc` file

Reply:
done